### ISSUE TITLE
Opening brace for test missing

### ISSUE THREAD

**Issue by JonCanning** [fsprojects/fantomas#806]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKBtAPAFUQZwBc8A%2BAXQB0A7AG0QLAPyLAF4qJHCAZAS0LHKRORQewhh0Y8QyYChTQWGBTpYAKIAPAA6JoBAHSIAjgFcAhtQYAnE4mu25Jyom27G8UZVVgAvitKQADTgUHh0BDyUAOZ4qBIhEKDekAAkBACeOnGQkQRBCeKpAG4WtnHoBdIADMHeEJAAkpRIlAQAylpm0IgAciYAtvl1YAAslWCkBT614kmqqRlZaDmtQ-MpJdRlaBXDAIwzyQAKZlGIAOo88AQAFmveewBMVeOT0tMFc9ILmcjLAEawWDUe6FDalP7xYaPQ7zNqIfo8ODUWCUACCBDUzQA8gAzXjOUHSXEWMKvKawr5gxaQyCA4FEqDgraQ3Z1ADMsO%2BHS6iAAQohcbArIgTlYzP06IgrIyODZEOT3pTxj8llB6SCuUzNtsoXURlr6jzugKhSKuLAAO7S6BmMJNIqwW0RVGy%2BwK7xvcQfaRU%2BppX7ZDWM4oQ8rjCAAVkNUGN-MFwsQAFUtDorLb7ZRHc6eK6YyTqGTPRTPiqA2q6UDNWWdayI2AAGwxyBx02JgDC1DteHbqMINj0wrdBaLqi9EB9szLNKDVZDzN1bO8AHZm62EyKALII-7S4ekj1jku%2B6eBgFzrWhlnh4YADjXnRNG8QvZRlH3hcP0nHvmVyXLtLBpeC51sMACcD68miuKMFYvb9P0ZhugQ8qKt6f7rDO54MsBtY3nUew1PWLaPvGZqIPCiLInm9Yjl%2B4g-pOiSnhWQE1mGOz1nsBzEXG0GwZRSLAjRwwobYaEThh3wAbOOHsdenH7DCxFNC0BDYpQ2BWOkly3B%2Bo7fseU7-lh6oXvJi5cZyvGkWiViwE48AACKINQPCIrByGocWSqliZZ5QLk854YpBEGsRm5mBoDS4tgNyIJQaifm0NzCgQlzXHcMYjC8PnoX5mEBSseS4RxeoPNGEVRU0uI8Bo2JpmYBDCpoWgingeC5u%2BMaRrlR6%2BSe-kVkFpUKeVqh7E2VUaAASrowrwBlenZX1hkDcZhXDaso2Wfsq7TXZ4rpNiVi8IQS1ZfWOUSb%2BBXSaZxXBWVS4Tfe01cHQfKRPAkRRBdbrXXlkl3dSRVsf%2BIXjdIewQRFJjUBEbnOHyKLQAA1ny4ro3QeAaW0EovsCAzdbRB43UxSgsYB5kQ899bPM2PSIJaSPxgQ1oJdgvwubVlA8C66LNNu-S7lYsT5mTQO3YNm3U3JtNjS90iPDxwwtihSIEJuCDIBLn7kyEpAzDkeAAGKdXEdEgD4QA)

**Please describe here fantomas problem you encountered**

#### Code

```fsharp
[<Tests>]
let tests =
  testList "tests"
    [
      test "test" {
        Expect.equal true true "unexpected"
      }
    ]
```
            

#### Result

```fsharp
[<Tests>]
let tests =
    testList "tests" [ test "test" Expect.equal true true "unexpected" } ]

```
            
#### Options

Fantomas Next - 4.0.0-alpha-001-1/1/1990

| Name | Value |
| ---- | ----- |
| `IndentSpaceNum` | `4` |
| `PageWidth` | `120` |
| `SemicolonAtEndOfLine` | `false` |
| `SpaceBeforeParameter` | `true` |
| `SpaceBeforeLowercaseInvocation` | `true` |
| `SpaceBeforeUppercaseInvocation` | `false` |
| `SpaceBeforeClassConstructor` | `false` |
| `SpaceBeforeMember` | `false` |
| `SpaceBeforeColon` | `false` |
| `SpaceAfterComma` | `true` |
| `SpaceBeforeSemicolon` | `false` |
| `SpaceAfterSemicolon` | `true` |
| `IndentOnTryWith` | `false` |
| `SpaceAroundDelimiter` | `true` |
| `MaxIfThenElseShortWidth` | `40` |
| `MaxInfixOperatorExpression` | `50` |
| `MaxRecordWidth` | `40` |
| `MaxArrayOrListWidth` | `40` |
| `MaxLetBindingWidth` | `40` |
| `MultilineBlockBracketsOnSameColumn` | `false` |
| `NewlineBetweenTypeDefinitionAndMembers` | `false` |
| `StrictMode` | `false` |

**Comment by nojaf** [comment]
Thanks for reporting this, sorry this slipped in.

**Comment by JonCanning** [comment]
Thanks, when will this be released? I could really use an ionide update to support this

**Comment by nojaf** [comment]
Well my plan was to release another alpha by the end of the week.
Once it is on NuGet, it can be updated in FsAutocomplete and then FsAutocomplete can be updated in Ionide-vs-code extension.

However, nothing prevents me from releasing this sooner.
In return, could you do the FsAutocomplete and Ionide part?

PS: I notice your original source contains:
```
[

]
```

You might be interested in the new setting `MultilineBlockBracketsOnSameColumn`.
This places the braces of a list on the same column. [For example](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKBtAPAFUQZwBc8A%2BAXQB0AncgOwBtECwD8iwBeK2iFwgGQCWhMOUi8iorjQgR0UmTPEixrUWGDyFMgKIAPAA6JoBAHSIAjgFcAhnWaVLie4%2BWWaiA0ZbxJ1aVoBfTQhSSAAacCg8RgIBGgBzPFQwdEiIUC0oABICAE9DZMg4gnC0mUgsgDdbR2TUzIgABgiGqABJGiQaAgBlfWtoRAA5SwBbUtaAFjKQsoCWmQytCryCtCLuieWqmuQ0eoaARgXMyAAFa3jEAHUBeAIACy3Mw4AmRpmwUjmT9U%2BV-J7KAAI1gsDoz3KOzotX2nwgr1%2BCkgPUQowEcDosBoAEECNpOgB5ABmgnckIUxNs0U%2B3wU8zKS2ROUBhVB4Ip2WqMKBB0yAGYkeU%2BgNEAAhRDE2CURAXSjWUaMRCUTk8ByIWk-Rn-FlrEFgiFCirc2EpeFgSZCiAo-qDCVSmV8WAAd2V0Gs0Q6lVg7ti2NVzg1mTpMgZCiZUNWQMg7MNOpNvPNAFYrVARXbJdLEABVfSGSjuz00b2%2BgT%2B1NUug04Na8M6qNsg2c427OrmgBsqZtovtWYAwnQPXg%2B9jCA5jNKA5Xq1oQxAw4t66z1rHm9DTXytAB2Lvp8WZmUAWTRwOVU%2BpQdntcXp110dXRvXidaAA5d7b9w7ECOsTRz1XLwUOcwAXdIlz1GMm0fBM21aABOd9RRxYkWEoEdRlGawAwIdVNXpX4I2tO9Gw5aDWzhVpDmac1uwzL9UXRTFy3NadAJkYDQL%2BW8GxXKD43Is1KOOGi92Q1CGIxcFmNaHDHDw0MCPA%2B8%2BNvGCKKOREaI6LoCEJGhsEoXJbkef8ZyA68wO45d9VI-ieVgo5BREj8cUoWA3HgAARRA6AEdFUOw3Ca3w7UrIg4o1zUwSjktGjD2sXQ2mJbAHkQGhtAAnoHmlAhbnuJ5U0mD5goU0Lth4qAIrI%2Bz1JeFM4oSjpiQEXRCXzawCGlPR9BlPA8DLP9UyTYqrxCuswujKq7I3c1Dk7BrdAAJSMaV4DykzCpG8yxpvcrrI2EpqpmyidwW1z5VyQlKEEQh1oK80ivk%2BdFImwoptUgTNwUQ43wWvhGDFOJ4DieI7oDR6SuesrmQqyDbI%2BmropeBC4ssOhYl89wxSxaAAGsxXlPHGDwPSegVb9wTGQbzVktjZh2yy9ogh9pufBp3i7IZEGdTH9wIV00uwQFvOamgBD9XFOmPUZT0oJIKwvJ6QJepnlPh7Yoq%2BmRXmE1oURwjECEPBBkAVgClYCSJSAWIo8AAMX65JWJAAIgA)

**Comment by JonCanning** [comment]
Sure

**Comment by nojaf** [comment]
@JonCanning I released https://www.nuget.org/packages/Fantomas/4.0.0-alpha-004.
To get this in Ionide:
* Dump the version in [FSAutocomplete](https://github.com/fsharp/FsAutoComplete/blob/master/paket.dependencies#L12).
* Afterwards: update the hash in ionide-vscode to the latest master commit of fsautocomplete. Similar to this [PR](https://github.com/ionide/ionide-vscode-fsharp/pull/1354/files).

Don't be worried when the CI fails on either one of those projects, that tends to happen from time to time.

@ErikSchierboom the fix for #785 is in there as well.

**Comment by ErikSchierboom** [comment]
@nojaf Awesome!
